includes/footer.php

```diff
@@ -1,21 +1,141 @@
-        <div class="body-container footer" style="">
+<div class="body-container footer" style="">
             <footer class="container">
                 <div class="row">
                     <h2 id="sponsor-title">At Top Gun Robotics, We Love Our Sponsors!</h2><br>
                     <hr style="margin-top: -7px; ">
                 </div>
-                
-                <div class="row" class="sponsors">
-                    <div class="col-sm-4">
-                         <img class="img-responsive center-block" id="sponsor-kenworth" alt="Kenworth Logo" src="http://nctrucking.wildapricot.org/Resources/Pictures/Kenworth%20Bar%20Logo%20with%20A%20PACCAR%20Company%20Black%20Trans%20Background%20800%20DPIpng.png"/>
-                    </div>
-                    <div class="col-sm-4">                        
-                       <img class="img-responsive center-block" id="sponsor-SAE" alt="SAE Northwest Logo" src="http://www.sae.org/images/sections/ms049/NWL_380839334_sae_nw_logo_smaller.gif"/>
-                    </div>
-                    <div class="col-sm-4">
-                        <img class="img-responsive center-block" id="sponsor-boeing" alt="Boeing Logo" src="http://www.iptv.org/medialib/media/Boeing_RGB_blue_std_P.JPG"/>
+                <style type="text/css">
+                    .left, .right {
+                        background: none !important;
+                        outline: 0;
+                    }
+
+                    .tech-slideshow {
+                        height: 300px;
+                        max-width: 5200px;
+                        margin: 0 auto;
+                        position: relative;
+                        overflow: hidden;
+                        transform: translate3d(0, 0, 0);
+                    }
+
+                    .tech-slideshow > div {
+                        width: 5200px;
+                        background: url('../../img/sponsors/sponsorSlidesSmall.jpg');
+                        position: absolute;
+                        top: 0;
+                        left: 0;
+                        height: 100%;
+                        transform: translate3d(0, 0, 0);
+                    }
+                    .tech-slideshow .mover-1 {
+                        animation: moveSlideshow 35s linear infinite;
+                    }
+                    .tech-slideshow:hover .mover-2 {
+                        opacity: 1;
+                    }
+
+                    @keyframes moveSlideshow {
+                        100% {
+                            transform: translateX(-100%);
+                        }
+                    }
+
+                    @media (min-width:600px) {
```

Comment:
I see your throwing in some magic numbers to detect desktop and mobile versions, bootstrap actually does this for us automatically. If you include your CSS in the `footer.less` file and compile it you can use bootstraps built in variables. You can see the variables we used in the file, and for all available ones check out bootstraps documentation http://getbootstrap.com/css/#responsive-utilities-classes